EditableTable/styles/DatasetStyles.css

```diff
@@ -89,11 +90,24 @@
   border-color: rgba(248, 58, 58, 0.326);
 }
 
+div[id^="DialogContainer__"][id$='_popupContainer_innerPopup'] {
```

Comment:
did you add this? I removed it because it was causing an issue, I think it got back when merging?